jdk/src/share/bin/jli_util.c

```diff
@@ -119,3 +119,77 @@ JLI_StrCCmp(const char *s1, const char* s2)
 {
    return JLI_StrNCmp(s1, s2, JLI_StrLen(s2));
 }
+
+JNIEXPORT JLI_List JNICALL
+JLI_List_new(size_t capacity)
```

Comment:
The definitions must be consistent with `jli_util.h` (remove `JNIEXPORT` and `JNICALL`).